crypto/account.go

```diff
@@ -321,12 +322,7 @@ func (lsa LogicSigAccount) IsDelegated() bool {
 func (lsa LogicSigAccount) Address() (addr types.Address, err error) {
 	hasSig := lsa.Lsig.Sig != (types.Signature{})
 	hasMsig := !lsa.Lsig.Msig.Blank()
-
-	// require at most one sig
-	if hasSig && hasMsig {
```

Comment:
this looked as a good early catch against malformed things

Reply:
Especially with addition of LMsig

Reply:
restored